### DISCUSSION TITLE
Tried to synchronously call a non-worklet function on the UI thread `pressable.js`

### DISCUSSION THREAD

**Discussion by midrizi** [nandorojo/moti#283] [Q&A] [answered]
@nandorojo Sorry to bother you

```
"react-native": "0.71.7"
"moti": "^0.25.1"
"react-native-reanimated": "~3.1.0"
```

<img width="250" src="https://user-images.githubusercontent.com/56638730/235328735-e39802d8-4cde-4729-b2b5-0d48ec7791e3.png" />

https://github.com/nandorojo/moti/blob/004b0dd0fa509668bd8707cd167d3b07291438ac/packages/moti/src/interactions/pressable/pressable.tsx#L91C1-L93

Is it supposed to be wrapped in `runOnJS`? Since doing so works.

**Comment by nandorojo** [comment, marked as answer]
please see the docs, you need to pass a worklet to `animate`

**Reply by midrizi** [reply]
Ahh.... prettier moved my 'worklet' at the top of the file. My bad.
